features/call/src/main/kotlin/io/element/android/features/call/ui/CallScreenPresenter.kt

```diff
@@ -166,6 +175,36 @@ class CallScreenPresenter @AssistedInject constructor(
         }
     }
 
+    @Composable
+    private fun HandleMatrixClientSyncState() {
+        val coroutineScope = rememberCoroutineScope()
+        DisposableEffect(Unit) {
```

Comment:
Do we really need the `DisposableEffect`? The coroutineScope might be enough

Reply:
We could manually stop the sync process when either we hang up the call or we receive a hang up message, but it seems safer to do it automatically when this is disposed.